commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/test/JunctionTests.java

```diff
@@ -141,6 +141,9 @@ public void testEvent() throws Exception {
         final FileObject real2 = baseDir.resolveFile("weakref.txt");
         real2.createFile();
         assertEquals("Weak Listener was abandoned", "Listener false true false", listener2.toString());
+
+        // Make sure we don't contaminate the fs for the next time the test runs
+        assertTrue(file2.delete());
```

Comment:
Hi @raboof 
If you want this to happen 100% of the time, as opposed to not if an assertion fails, you must put it in a `finally` block.